docs/translations.rst

```diff
@@ -6,6 +6,38 @@ localization strings. For more comprehensive information, consult the
 `Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_
 or `this helpful blog post <http://blog.bessas.me/post/65775299341/using-gettext-in-django>`_.
 
+How translations work
+---------------------
+There are three major components to the translations system - identifying
+strings that need to be translated, having human translators write translations
+for those strings, and then actually inserting those translations when
+appropriate.
+
+1. ``./manage.py makemessages`` scans the source code and pulls out strings
+   that have been tagged for translation. It does not actually execute code, so
+   you really can't do anything fancy in these strings.
```

Comment:
Maybe this?

```suggestion
   you must not format the string _inside_ the translation lookup function (i.e.
   never use f-strings or call `.format()` inside `gettext(...)`)
```

Reply:
Explicitly "prohibiting" f-strings here would clarify the entire doc, and I'd walk back my _"could be either that or an f-string"_ statement.

Reply:
I don't like my suggestion anymore after seeing your comment above :joy: